### PR TITLE
docs(cli): add docs for lb4 openapi command

### PR DESCRIPTION
**Pull request by raymondfeng**
Add docs for `lb4 openapi`.

## Checklist

- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [x] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

### CODE REVIEW

File: docs/site/OpenAPI-generator.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+Generates artifacts from an OpenAPI spec into a LoopBack application.
+
+```sh
+lb4 openapi [<url>] [options]
+```
+
+### Options
+
+- `--url`: URL or file path of the OpenAPI spec
+- `--validate`: Validate the OpenAPI spec Default: false
```
Comment (virkt25) [inline docs/site/OpenAPI-generator.md]: Should there be a `.` after `spec`?

File: docs/site/OpenAPI-generator.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+
+### Arguments
+
+`<url>`: URL or file path of the OpenAPI spec Type: String Required: false.
```
Comment (virkt25) [inline docs/site/OpenAPI-generator.md]: Same here. I think we should have a period `.` after `spec` and `String`

File: docs/site/OpenAPI-generator.md [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+    return_message?: string;
+  }> {
+    throw new Error('Not implemented');
+  }
+}
+```
+
+### OpenAPI Examples
+
+Try out the following specs with `lb4 openapi`:
```
Comment (virkt25) [inline docs/site/OpenAPI-generator.md]: My first thought was that this command is limited to the following specs only. Might be better to say `try out the following specs or use your own with `lb4 openapi`.

**Comment by raymondfeng** [comment]
@virkt25 Fixed. PTAL.

**Pull request merged**